### PR TITLE
Add codes starting with 50

### PR DESCRIPTION
**Pull request by renetapopova**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-197.surge.sh

Updated pages:
- [List of all GQLSTATUS server error codes](https://neo4j-docs-status-codes-197.surge.sh/status-codes/5/errors/gql-errors)

File: modules/ROOT/pages/errors/gql-errors.adoc [hunk 1]
```diff
@@ -432,3 +432,37 @@ Status description:: error: syntax error or access rule violation - auth info ex
...
+=== 50N11
+
+
+Status description:: error: general processing exception - constraint creation failed. Unable to create `{ $constrDescrOrName }`.
+
+=== 50N12
+
+Status description:: error: general processing exception - constraint drop failed. Unable to drop `{ $constrDescrOrName }`.
+
+=== 50N42
```
Comment (Lojjs) [inline modules/ROOT/pages/errors/gql-errors.adoc:466]: Doesn't need to be part of this PR, but we probably want to have some more info written down here about this being the default code and you should not depend on it etc. Maybe we even want to link to it from the query log docs?

**Review by Lojjs — APPROVED** [review]
Just a point for later, everything in this one looks correct otherwise

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-197.surge.sh

Updated pages:
- [List of all GQLSTATUS server error codes](https://neo4j-docs-status-codes-197.surge.sh/status-codes/5/errors/gql-errors)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-status-codes-197.surge.sh

Updated pages:
- [List of all GQLSTATUS server error codes](https://neo4j-docs-status-codes-197.surge.sh/status-codes/5/errors/gql-errors)

**Pull request merged**